### ISSUE TITLE
P25Hosts.ini is picky about field delimiters.

### ISSUE THREAD

**Issue by N4IRS** [g4klx/P25Clients#2]
I am seeing a issue with the field delimiters in the host file. Below is the host file with TAB delimiters
My P25Gateway.ini is set to startup 10400 for this test.

# P25Hosts.txt
#
# The format of this file is the number of the Talk Group followed by the host name or address and port
#
# World Wide, 1010x
10100   85.119.82.151   41000

# North America, 1020x
10200   45.62.252.188   41000

# Europe, 1030x
10300   176.9.1.168     41000

# Pacific, 1040x
10400   125.63.57.138   41000

root@mmdvm:/opt/P25Gateway# ./P25Gateway P25Gateway.ini
I: 2016-10-17 18:35:43.625 Opening P25 network connection
I: 2016-10-17 18:35:43.627 Opening P25 network connection
I: 2016-10-17 18:35:43.630 Loaded 5 P25 reflectors
I: 2016-10-17 18:35:44.151 Loaded 44607 Ids to the callsign lookup table
M: 2016-10-17 18:35:44.153 Starting P25Gateway-20161011
M: 2016-10-17 18:35:44.156 Linked at startup to reflector 10400
I: 2016-10-17 18:35:44.158 Started the DMR Id lookup reload thread

Notice P25Gateway is reporting 5 reflectors. There are 4 in the file and it linked to startup 10400.

I replaced the TABs between the fields for 10400 with spaces

root@mmdvm:/opt/P25Gateway# ./P25Gateway P25Gateway.ini
I: 2016-10-17 18:36:49.323 Opening P25 network connection
I: 2016-10-17 18:36:49.325 Opening P25 network connection
I: 2016-10-17 18:36:49.328 Loaded 4 P25 reflectors
I: 2016-10-17 18:36:49.891 Loaded 44607 Ids to the callsign lookup table
M: 2016-10-17 18:36:49.894 Starting P25Gateway-20161011
I: 2016-10-17 18:36:49.896 Started the DMR Id lookup reload thread

Now reporting 4 reflectors and not connecting at startup to 10400

P25Gateway seems to stop processing if it does not like the delimiter.

73, Steve N4IRS

**Comment by g4klx** [comment]
It should be a little less picky now, spaces can be used also.

**Comment by phl0** [comment]
I found another issue in P25Gateway leading to the fact that the log always says that it loaded one more reflector than really alive in the P25Hosts.txt. The code counts the parrot as extra reflector. Thus 4 reflectors in the Hosts and an active parrot result in the log saying that it loaded 5 reflectors. 

I created a PR to correct this. With this patch it should log the real number of reflectors and an extra line if the parrot is enabled. See https://github.com/g4klx/P25Clients/pull/3
